rest/tests/test_rest_authorization.py

```diff
@@ -642,3 +648,234 @@ def test_members_delete(self):
             self.delete_member(urlfor='groups', repo_slug=self.repo.slug,
                                username=self.author_user.username,
                                group_type=group_type)
+
+    def test_resource_get(self):
+        """Test retrieve learning resource"""
+        self.assertEqual(
+            1, self.get_learning_resources(self.repo.slug)['count'])
+
+        tarball_file = self.get_course_single_tarball()
+        import_file(
+            tarball_file, self.repo.id, self.user.id)
+        resource = get_resources(self.repo.id).first()
+        resource.static_assets.add(StaticAsset.objects.first())
+        lr_id = self.get_learning_resources(self.repo.slug)['results'][0]['id']
+
+        # author_user has view_repo permissions
+        self.logout()
+        self.login(self.author_user.username)
+        self.assertEqual(
+            6, self.get_learning_resources(self.repo.slug)['count'])
+        self.get_learning_resource(self.repo.slug, lr_id)
+
+        # user_norepo has no view_repo permission
+        self.logout()
+        self.login(self.user_norepo.username)
+        self.get_learning_resources(
+            self.repo.slug, expected_status=HTTP_403_FORBIDDEN)
+        self.get_learning_resource(
+            self.repo.slug, lr_id, expected_status=HTTP_403_FORBIDDEN)
+
+        # as anonymous
+        self.logout()
+        self.get_learning_resources(
+            self.repo.slug, expected_status=HTTP_403_FORBIDDEN)
+        self.get_learning_resource(
+            self.repo.slug, lr_id, expected_status=HTTP_403_FORBIDDEN)
+
+    def test_resource_create(self):
```

Comment:
This test should probably be named `test_resource_post`, since it's not testing creation.